src/main/java/cms/gov/madie/measure/services/QdmPackageService.java

```diff
@@ -96,6 +96,39 @@ public byte[] getQRDA(QrdaRequestDTO qrdaRequestDTO, String accessToken) {
     }
   }
 
+  @Override
+  public String getHumanReadable(Measure measure, String username, String accessToken) {
```

Comment:
should we restrict for draft only?

Reply:
We have a follow up story to allow versioned measures as well. I think we should include it within this method to check for versioned or draft rather than separate methods based on measure status.